srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -156,36 +157,49 @@ private static String translateFromVista(final String vistaField)
     
     private void parseWeightResults(final Patient patient, final List<String> weightResults) throws ParseException
     {
-    	// The last entries are the most recent so we use those.
-    	// Get the most recent weight measurement within the already specified range.
-    	final String[] weightLineTokens = weightResults.get(weightResults.size()-2).split("[\\s\\^]+");
+    	/* The last entries are the most recent so we use those.
```

Comment:
The spacing here is a little out of whack (spaces and tabs). Also, put a '-' after the first asterisk to mark this as a preformatted comment. (Or else Eclipse might reformat it.) Like this:

    /*-
     * A B C D
     * 1 2 3 4
     */

The comment is very helpful, though.